### DISCUSSION TITLE
开源共同体从科学共同体中继承了什么--文化视角 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#18] [Announcements]
# 开源共同体从科学共同体中继承了什么--文化视角 - 开源之道

https://opensourceway.community/posts/book-of-open-source/what-is-os-community-got-from-sciense-community/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
这其实已经开始摸到 开源之史 的创作思路了,
一切当今现象/行为, 都不是无根之水, 都是有其思想源头的,
即便是当事人本身没有意识到, 但是, 修史就值得划清谱系, 以便同理预见变化;

这点可以参考 科学的历程, 作者20多年来4次增订的变化思路,
既然科学是全人类共有/共享/共建/共管的,

开源也不过是科学的一个侧面,
甚至于, 在没有计算机前, 开源协作就早已在其它学科中开展了,
比如哲学/数学/...如果没有全球开放协作, 很难想象一个公理/分子式 如何快速变成每个研究室的基本元素;

所以, 俺感觉不是从科学共同体中继承了什么,
而是在科学共同体中复用了什么,

更加值得讨论的是, 开源共同体 比科学共同体 又发展出什么新的共同行为/文化/准则/思想/...

PS:
科学史必须同时从技术/人文/社会/...等方面同时记录,
开源史也一样, 而且多出经济史的对照, 挑战更大,
如何将越来越分化的学科角度, 重新用一种自然的陈述形式,
统一在一起, 回顾FLOSS/自由/开源的高速发展?
